src/darwin/Framework/CHIP/templates/MTRClusters-src.zapt

```diff
@@ -53,6 +65,7 @@ using chip::SessionHandle;
 {{/unless}}
 - (void){{asLowerCamelCase name}}WithParams: (MTR{{asUpperCamelCase parent.name}}Cluster{{asUpperCamelCase name}}Params * {{#unless (commandHasRequiredField .)}}_Nullable{{/unless}})params expectedValues:(NSArray<NSDictionary<NSString *, id> *> *)expectedValues expectedValueInterval:(NSNumber *)expectedValueIntervalMs completion:({{>command_completion_type command=.}})completion
 {
+    NSString * logPrefix = [NSString stringWithFormat:@"MTRDevice command %u %" PRIu16 " %u %u", self.device.deviceController.fabricIndex, _endpoint, (unsigned int)MTRClusterIDType{{asUpperCamelCase parent.name}}ID, (unsigned int)MTRCommandIDTypeCluster{{asUpperCamelCase parent.name}}Command{{asUpperCamelCase name}}ID];
```

Comment:
Hrm.  I forgot we have a lint for PRIu16.... This should probably just be %u.  Passing a uint16_t to %u should work correctly.

Reply:
Fixed in latest change